src/components/config-pages/local-footer.tsx

```diff
@@ -88,13 +90,15 @@ const LocalFooter = ({
               </ul>
             }
 
+            <Wysiwyg html={suLocalFootPrCo?.processed} className="text-archway"/>
+
             {suLocalFootSocial &&
-              <ul className="list-unstyled flex gap-2">
+              <ul className="rs-mt-4 list-unstyled flex gap-11 items-center children:mb-0">
```

Comment:
Why `children:mb-0` instead of `mb-0` on the `<li>` element? I'm not asking for any changes, just curious of your choice here.

Reply:
Ah, this was user error 😄 I was moving some things around and added seperate `<li>` item so I added `children:mb-0` so that all children are targeted. I eventually removed it and forgot to rework the classes. In this case, we can definitely further specify and add `className="children:mb-0"` to the `<li>`

Reply:
This comment not related to whether we use children:mb-0 on ul or mb-0 on li, but `*:` has officially replaced `children:` - it's in Tailwind core now 😄 @rebeccahongsf